### DISCUSSION TITLE
Mapping casi d'uso: Sezioni / Campi ripetuti

### DISCUSSION THREAD

**Discussion by biadanpad** [italia/ansc#837] [Miglioramenti] [answered]
Buonasera, si segnala che diversi file di mapping dei casi d'uso contengono Sezioni / Campi ripetuti spesso aventi dati discordanti nei campi Obbligatorio / Note obbligatorieta' formule / Condizioni obbligatorieta'

Segnaliamo ad esempio:
Trascr_006
Rico_003
Sciogl_UnCiv_003
Sep_Div_001
Sep_Div_002
Sep_Div_008

**Comment by gbuemi** [comment]
Buongiorno,
abbiamo preso in carico la issue.

Un saluto,
Supporto ANSC

**Comment by gbuemi** [comment]
Salve @biadanpad,
potete farci per favore un esempio di sezioni / campi ripetuti con dati discordanti? Per esempio Trascr_006 riga 11.

Un saluto e grazie,
Supporto ANSC

**Comment by MarcoSalaPad** [comment]
Salve,
ad oggi questa issues non ha più avuto un seguito.

Sinceramente ci sembrava già abbastanza dettagliato l'elenco dei casi d'uso dove avevamo riscontrato la presenta di più sezioni/campi ripetuti.

Ad esempio abbiamo appena riscontrato, per il caso d'uso Trascr_012, alcuni campi sono ripetuti ben 5 volte nella sezione Dati Generali (TipoRichiesta, Data richiesta trasmissione...).

E sono tutti obbligatori sul tracciato del caso d'uso.

**Comment by gbuemi** [comment]
Buongiorno @MarcoSalaPad,
vi informiamo che stiamo predisponendo un documento che chiarisce come leggere la documentazione fornita. 
 
Un saluto,
Supporto ANSC

**Comment by MarcoSalaPad** [comment]
Buonasera @gbuemi,
a distanza di quasi due mesi non abbiamo informazioni in merito a questa issue.
Eravamo in attesa del documento che ci avevate indicato come chiarimento nella lettura della documentazione.

E' fondamentale avere queste vostre specifiche per pianificare lo sviluppo dei casi d'uso da implementare e soprattutto per la manutenzione di quelli già in uso dai clienti.

Restiamo in attesa urgente di questo documento.

Un saluto.

**Comment by gbuemi** [comment]
Buongiorno @MarcoSalaPad,
ci scusiamo per la mancata condivisione del documento in questione. Abbiamo dato una bassa priorità rispetto ai vari tasks di sviluppo perché nessuna delle SH attualmente in produzione ci ha segnalato questo problema e di conseguenza per nessuna SH è risultato bloccante. 

A stretto giro rilasceremo un documento che chiarisce come leggere i mapping dei vari casi d'uso.

Un saluto,
Supporto ANSC

**Comment by gbuemi** [comment, marked as answer]
Buongiorno @MarcoSalaPad,
vi informiamo che con la versione [1.28.4](https://github.com/italia/ansc/blob/main/docs/Changelog.md) è stata pubblicata la [Guida alla documentazione del file di mapping dei casi d'uso](https://github.com/italia/ansc/blob/main/docs/Mapping_casi_uso/readme_mapping.md).

Un saluto,
Supporto ANSC
